src/entities/Place/model.test.ts

```diff
@@ -177,7 +177,7 @@ describe(`findWithAggregates`, () => {
           AND p.base_position IN (
             SELECT DISTINCT(base_position) FROM "place_positions" WHERE position IN ($1)
           )
-        ORDER BY p.like_rate DESC, p."deployed_at" desc
```

Comment:
Why would we test both cases if like_rate is no longer used for endpoints or find queries? The field won't be removed and it will be up to date always but for queries, it's not used anymore.